src/nnfusion/core/kernels/cuda_gpu/cuda_emitter.hpp

```diff
@@ -198,26 +174,27 @@ namespace nnfusion
                 virtual bool require_cublas_handle() { return false; }
             };
 
-            class AntaresCudaKernelEmitter : public BlockCudaEmitter
+            class AntaresCudaKernelEmitter : public CudaEmitter
```

Comment:
AntaresCudaKernelEmitter could be BlockCudaEmitter because BlockFusion pass has processed it.